test/es6module/bug_issue_3257.js

```diff
@@ -0,0 +1,30 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.RegisterModuleSource("mod0.js", `

```

Comment:
Looks like this isn't correctly registering as a module, since a test is failing on CI:
```
>Error in opening file 'D:\j\workspace\x86_test_prtest3d272688\test\es6module\mod0.js' : The system cannot find the file specified.
```

Reply:
Interesting this didn't show up with other test cases.